packages/browser/src/browser/index.ts

```diff
@@ -353,7 +353,16 @@ async function loadAnalytics(
   const plugins = settings.plugins ?? []
 
   const classicIntegrations = settings.classicIntegrations ?? []
-  Stats.initRemoteMetrics(legacySettings.metrics)
+
+  const segmentLoadOptions = options.integrations?.['Segment.io'] as
```

Comment:
But I think when loading and passing in the integrations object directly instead of through CDN settings we don't require Segment.io

https://github.com/segmentio/analytics-next/blob/master/packages/core/src/events/interfaces.ts#L24-L27

Reply:
Ah, that makes sense